.bin/phraseapp/const.rb

```diff
@@ -8,6 +8,7 @@ module Const
   PHRASEAPP_PROJECT_ID = '9036e89959d471e0c2543431713b7ba1'
   PHRASEAPP_FALLBACK_LOCALE = 'en_US'
   PHRASEAPP_TAG = 'oxid'
+  LOCALE_PREFIX = 'wdpg_'
```

Comment:
did we already agree on the prefix? our controllers are prefixed with `wcpd_` maybe we should use the same prefix for the strings?

Reply:
Also @nickstamat can we create an .env variable to have both the same values here in ruby and in the `Language` override?

Reply:
we have to change it, as soon as we get the registered prefix

Reply:
we won't change where the prefix is taken from for now